README.md

```diff
@@ -25,11 +27,7 @@ The default locations for the config file is `~/.hyperkube-config.yaml`. You can
 ## Post cluster and creds to hyperkube store
 
 ```bash
-# with config file in default location ~/.hyperkube-config.yaml
 kubectl hyperkube add --k8s-config ~/.kube/config
-# with config file in default location ~/location2/.hyperkube-config.yaml
-# kubectl hyperkube -c ~/location2/.hyperkube-config.yaml add --k8s-config ~/.kube/config
-# kubectl hyperkube --config ~/location2/.hyperkube-config.yaml add --k8s-config ~/.kube/config
```

Comment:
I removed this section since it is documented nicely in the section above. Looks cleaner without the examples i think